content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/analyze-query-plan.md

```diff
@@ -0,0 +1,786 @@
+|               |   UnionExec                                                              |
+|               |     SortExec: expr=[city@0 ASC NULLS LAST,time@2 DESC]                   |
+|               |       ParquetExec: file_groups={...}, projection=[city, min_temp, time]  |
+|               |     SortExec: expr=[city@0 ASC NULLS LAST,time@2 DESC]                   |
+|               |       ParquetExec: file_groups={...}, projection=[city, min_temp, time]  |
+|               |                                                                          |
+```
+
+{{% caption %}}
+Output from `EXPLAIN SELECT city, min_temp, time FROM h2o ORDER BY city ASC, time DESC;`
+{{% /caption %}}
+
+Each step, or _node_, in the physical plan is an `ExecutionPlan` name and the key-value _expressions_ that contain relevant parts of the query--for example, the first node in the [`EXPLAIN` report](#explain-report) physical plan is a `ParquetExec` execution plan:
+
+```text
+ParquetExec: file_groups={...}, projection=[city, min_temp, time]
+```
+
+Because `ParquetExec` and `RecordBatchesExec` nodes retrieve and scan data in InfluxDB queries, every query plan starts with one or more of these nodes.
+
+#### Physical plan data flow
+
+Data flows _up_ in a query plan.
+
+The following diagram shows the data flow and sequence of nodes in the [`EXPLAIN` report](#explain-report) physical plan:
+
+```text
+                   ┌─────────────────────────┐
+                   │ SortPreservingMergeExec │
+                   └─────────────────────────┘
+                                ▲
+                                │
+                   ┌─────────────────────────┐
+                   │        UnionExec        │
+                   └─────────────────────────┘
+                                ▲
+             ┌──────────────────┴─────────────────┐
+             │                                    │
+┌─────────────────────────┐          ┌─────────────────────────┐
+│        SortExec         │          │        SortExec         │
+└─────────────────────────┘          └─────────────────────────┘
+             ▲                                    ▲
+             │                                    │
+             │                                    │
+┌─────────────────────────┐          ┌─────────────────────────┐
+│       ParquetExec       │          │       ParquetExec       │
+└─────────────────────────┘          └─────────────────────────┘
+```
+
+{{% caption %}}
+Execution and data flow in the [`EXPLAIN` report](explain-report) physical plan.
+`ParquetExec` nodes execute in parallel and `UnionExec` combines their output.
+{{% /caption %}}
+
+The following steps summarize the [physical plan execution and data flow](#physical-plan-data-flow):
+
+1. Two `ParquetExec` plans, in parallel, read data from Parquet files.
+    Each `ParquetExec` node reads data sequentially from each file in its file group, and then outputs a stream of data to its corresponding `SortExec` node.
```

Comment:
If you want to make it clearer, each `ParquetExec` can include one or many groups, each group can have one or many files. Different groups will read in parallel but files in each group will be read sequentially